app/src/main/java/appricottsoftware/clarity/sync/ClarityClient.java

```diff
@@ -76,7 +72,7 @@ public void authenticateUser(String email, String password, JsonHttpResponseHand
 
             JSONObject jsonParams = new JSONObject();
             try {
-                client.setMaxRetriesAndTimeout(1, 1000);
+                client.setMaxRetriesAndTimeout(0, 1000);
```

Comment:
Why did you change it from 1 to 0?

Reply:
I feel it takes too long to get a toast saying the server is down when there's 2 attempts to reach it. If everyone's content with 1 retry I can put it back (:

Reply:
Ahhh I understand now!